packages/@aws-cdk/aws-apigatewayv2-integrations/lib/http/eventbridge.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  HttpRouteIntegrationBindOptions,
+  HttpRouteIntegrationConfig,
+  EventBridgeIntegrationRequestParameters,
+  AwsServiceIntegrationSubtype,
+} from '@aws-cdk/aws-apigatewayv2';
+import { PolicyStatement, Role, ServicePrincipal } from '@aws-cdk/aws-iam';
+import { IEventBus } from '../../../aws-events';
+import { AwsServiceIntegration } from './aws/integration';
+import { AwsServiceIntegrationOptions } from './base-types';
+
+/**
+ * Properties to initialize `EventBridgeIntegration`.
+ */
+export interface EventBridgeIntegrationProps extends AwsServiceIntegrationOptions {
+  /**
+   * The EventBridge API call to proxy to.
+   * @default AwsServiceIntegrationSubtype.EVENT_BRIDGE_PUT_EVENTS
+   */
+  readonly integrationSubtype?: AwsServiceIntegrationSubtype;
```

Comment:
modeling: it does not make sense for customers to configure any other type here besides event bridge supported APIs.

So I would suggest creating a specific type here.

```ts
  readonly targetAction: EventBridgeTargetAction;
```